kernel/process/process.rs

```diff
@@ -207,6 +207,11 @@ impl Process {
         self.pid
     }
 
+    /// The thread ID
+    pub fn tid(&self) -> PId {
+        self.pid
```

Comment:
As you said in a comment, Could you leave a comment why using `self.pid` is OK for now (`Single-threaded process ...`)?

Reply:
Done